solved_exercises/dani/02-tuples.py

```diff
@@ -0,0 +1,43 @@
+import itertools
+'''*********************************************************************************************'''
+print('*****************Check wether an element exists within a tuple******************')
+my_tuple = 1, 2, 3, 3, 4, 5, 6
+print('my_tuple is', my_tuple)
+a=3
+b=7
+print('Version1')
+print (a in my_tuple and (a," is in my_tuple") or (a," isn't in my_tuple"))
+print (b in my_tuple and (b," is in my_tuple") or (b," isn't in my_tuple"))
+'''---------------------------------------------------------------------------'''
+print('Version2')
+print(my_tuple.count(a) and (a," is in my_tuple") or (a," isn't in my_tuple"))
+print(my_tuple.count(b) and (b," is in my_tuple") or (b," isn't in my_tuple"))
+
+'''*********************************************************************************************'''
+print('***************Sum up all the number elements within a tuple********************')
+my_tuple = 1, 2, 3, 3, 4, 5, 6
+print("Version 1: sum of the numbers in my_tupple", my_tuple, "is: ",sum(my_tuple))
+'''---------------------------------------------------------------------------'''
+my_tuple = 1,'2', 'a', 'asd2', 4, 5, 'a6'
+suma = [tupla for tupla in my_tuple if isinstance(tupla,int)]
+print("Version 2: sum of the numbers in my_tupple ", my_tuple, "is: ", sum(suma))
+
+'''*********************************************************************************************'''
+print('**********************Find the repeated items of a tuple*******************')
```

Comment:
Maybe the exercise requirement was too succint. You have wrote some code that counts how many times hard-coded values are found in the tuple. The goal of the exercise would be to find all repeated items in a tuple.

Reply:
OK, I'll re do the exercises